src/NuGet.Core/NuGet.Frameworks/CompatibilityProvider.cs

```diff
@@ -181,18 +182,18 @@ private bool IsCompatibleWithTarget(NuGetFramework target, NuGetFramework candid
 
         private static bool IsCompatibleWithTargetCore(NuGetFramework target, NuGetFramework candidate)
         {
-            if (target.IsNet5Era)
-            {
-                return NuGetFramework.FrameworkNameComparer.Equals(target, candidate)
-                    && IsVersionCompatible(target.Version, candidate.Version)
-                    && (!candidate.HasProfile || StringComparer.OrdinalIgnoreCase.Equals(target.Profile, candidate.Profile));
-            }
-            else
-            {
-                return NuGetFramework.FrameworkNameComparer.Equals(target, candidate)
```

Comment:
If the target.IsNetEra == false, previously we used to return line 192 above.

With this change the result is changed to the result below(line 185), is this intentionally? Was a bug there before?

Reply:
Can you clarify? I refactored this so the net5 bit was split off into its own section. The logic *should* be about the same, except for the Platform parts.

Reply:
I misunderstood the change. I read it again and all good.